tests/e2e/stepDefinitions/createSpaceContext.js

```diff
@@ -0,0 +1,58 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const { CreateSpacePage } = require("../pageObjects/CreateSpacePage.js");
+
+const assert = require("assert");
```

Comment:
use `expect` from playwright for assertion
```suggestion
import { expect } from '@playwright/test';
```

Reply:
This can be done in other refactoring PRs